packages/firebase_admob/example/ios/Runner/AppDelegate.m

```diff
@@ -3,14 +3,65 @@
 // found in the LICENSE file.
 
 #include "AppDelegate.h"
+#import "FLTFirebaseAdMobPlugin.h"
 #include "GeneratedPluginRegistrant.h"
 
-@implementation AppDelegate
+@interface NativeAdFactoryExample : NSObject <FLTNativeAdFactory>
+@end
+
+@implementation NativeAdFactoryExample
+- (GADUnifiedNativeAdView *)createNativeAd:(GADUnifiedNativeAd *)nativeAd
+                             customOptions:(NSDictionary *)customOptions {
+  // Create and place ad in view hierarchy.
+  GADUnifiedNativeAdView *adView =
+      [[NSBundle mainBundle] loadNibNamed:@"UnifiedNativeAdView" owner:nil options:nil].firstObject;
+
+  // Associate the native ad view with the native ad object. This is
+  // required to make the ad clickable.
+  adView.nativeAd = nativeAd;
+
+  // Populate the native ad view with the native ad assets.
+  // The headline is guaranteed to be present in every native ad.
+  ((UILabel *)adView.headlineView).text = nativeAd.headline;
+
+  // These assets are not guaranteed to be present. Check that they are before
+  // showing or hiding them.
```

Comment:
That this depends on the ad that is displayed?

Reply:
Yes this depends on the information that is provided in the `UnifiedNativeAd`. Some ads may have this item and others won't.

Reply:
I see. Suggestion: what about we add a link to https://developers.google.com/admob/ios/native/advanced in a comment?